package.json

```diff
@@ -2,19 +2,20 @@
   "name": "itowns",
   "version": "0.0.0-alpha",
   "description": "A JS/WebGL framework for 3D geospatial data visualization",
-  "main": "dist/itowns.js",
+  "main": "lib/Main.js",
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/*",
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint && npm run build && npm run doclint",
     "build": "webpack -p",
     "start": "webpack-dev-server -d --inline --hot",
-    "prepublish": "npm run build"
+    "prepublish": "npm run lint && npm run doc && npm run build && BABEL_DISABLE_CACHE=1 babel src --out-dir lib"
```

Comment:
> Where does this (npm run doc) generate the documentation?

I thought it would generate it in `doc`, but actually it does it in `out`, except in `autodeploy.sh` for publishing to gh_pages.

So we don't actually package it. I'm not sure we need to, so I'm going to remove this `npm run doc` from the prepublish phase.

> shouldn't it rather be npm run doclint instead (only to ensure the jsdoc is OK)?

it is already run in the `test` phase (so run for each PR).  If we don't package the doc, I guess we don't need to lint it there.

Thanks for the remark about the doc, I now thing we should just do:
`npm run build && BABEL_DISABLE_CACHE=1 babel src --out-dir lib`. 

WDYT?